lib/api/config.rb

```diff
@@ -0,0 +1,6 @@
+require "config"
+
+module Api
+  CONFIG_PATH = File.expand_path(File.join("..", "..", "..", "config", "api.yml"), __FILE__).freeze
```

Comment:
I can do that. I just wasn't sure, with this now being in /lib, if I should assume that `Rails` is around?